projects/dotcms-field-elements/src/components/dot-binary-file/dot-binary-file.tsx

```diff
@@ -169,9 +190,9 @@ export class DotBinaryFileComponent {
     @Listen('drop', { passive: false })
     HandleDrop(evt: DragEvent): void {
```

Comment:
`The function HandleDrop has a cyclomatic complexity of 4 which is higher than the threshold of 3 (cyclomatic-complexity)tslint(1)`